SignalShareExtension/Info.plist

```diff
@@ -19,31 +19,74 @@
 	<key>CFBundleShortVersionString</key>
 	<string>2.20.0</string>
 	<key>CFBundleVersion</key>
-	<string>2.20.0.0</string>
+	<string>2.20.0.1</string>
+	<key>ITSAppUsesNonExemptEncryption</key>
```

Comment:
Is this key-value necessary?  The main app uses it.

Reply:
I'm not sure. The key only exists to pass AppStore review. Submitting it to the AppStore should answer that for us.